src/core/dbus-execute.c

```diff
@@ -3291,6 +3293,8 @@ int bus_exec_context_set_transient_property(
                                 dirs = &c->exec_paths;
                         else if (streq(name, "NoExecPaths"))
                                 dirs = &c->no_exec_paths;
+                        else if (streq(name, "ExtensionDirectories"))
+                                dirs = &c->extension_directories;
```

Comment:
Hmm, there is no validity check...

Reply:
Same as other directory-based parameters, there's a minimal parsing and validity checks done by the parser (absolute path, etc), but beyond that it's checked at use-time, not at configure-time

Reply:
Oops. Each string is checked in the above. Sorry for the noise. Please forget.